app/assets/stylesheets/common/base/search-menu.scss

```diff
@@ -267,6 +267,19 @@
       .search-item-prefix {
         margin-right: 0.5em;
       }
+
+      .search-item-tag {
+        color: var(--primary-high);
+        font-size: var(--font-down-1);
+      }
+
+      .d-icon-tag {
+        // match category badge styling
+        // tag/category suggestions can be displayed simultaneously
+        font-size: var(--font-down-2);
+        margin-right: 4px;
+        margin-left: -1px;
```

Comment:
I'm always a bit leery of negative margins. Is there no other way to do it?

Reply:
It's fine without it: 

<img width="543" alt="image" src="https://user-images.githubusercontent.com/368961/129921122-39aef07c-7c53-402c-957f-2253f02ea26e.png">